test/shell/test_misc.sh

```diff
@@ -5,13 +5,17 @@ dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 runner=$(get_test_runner "${1:-local}")
 
 test_disappearing_class() {
+  tmp=$(mktemp -d -t test_disappearing_class-XXXXXXXXXX)
+  out=$tmp/out
+  cache=$tmp/cache
   git checkout test_expect_failure/disappearing_class/ClassProvider.scala
-  bazel build test_expect_failure/disappearing_class:uses_class
+  bazel --output_base=$out build --disk_cache=$cache test_expect_failure/disappearing_class:uses_class
```

Comment:
I think that `--disk_cache=` can be simpler (empty value simply disables disk cache)

Reply:
Also - I wonder if we can do without changing the output_base...